pbs_light.go

```diff
@@ -693,8 +693,10 @@ func serve(revision string, cfg *config.Configuration) error {
 		glog.Fatalf("Failed to create the bidder params validator. %v", err)
 	}
 
+	bidderInfos := adapters.ParseBidderInfos("./static/bidder-info", openrtb_ext.BidderList())
```

Comment:
This _should_ be safe, but... please do build & run this once in a docker container locally, just to make sure the files & paths still work.

Reply:
This is the exact same line as from below, just moved up so `exchange.NewExchange` has access to it. Is there something about moving it that might break something?

Reply:
ah, of course... no, you're fine then. Ignore me